src/test/java/team/easytravel/model/listmanagers/packinglistitem/ItemCategoryTest.java

```diff
@@ -0,0 +1,79 @@
+package team.easytravel.model.listmanagers.packinglistitem;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+class ItemCategoryTest {
+    @Test
+    public void constructorNullThrowsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new ItemCategory(null));
+    }
+
+    @Test
+    public void constructorInvalidCategoryThrowsIllegalArgumentException() {
+        String invalidCategory = "";
+        assertThrows(IllegalArgumentException.class, () -> new ItemCategory(invalidCategory));
+    }
+
+    @Test
+    public void isValidItemCategory() {
+        // null category
+        assertThrows(NullPointerException.class, () -> ItemCategory.isValidItemCategory(null));
+
+        // blank category
+        assertFalse(ItemCategory.isValidItemCategory("")); // Empty string
+        assertFalse(ItemCategory.isValidItemCategory(" ")); // Spaces only
+
+        // invalid category
+        assertFalse(ItemCategory.isValidItemCategory("^")); // Non-alphanumeric characters
+        assertFalse(ItemCategory.isValidItemCategory("Dog*")); // Contains non-alphanumeric characters.
+        assertFalse(ItemCategory.isValidItemCategory(
+                "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")); // Contains 34 characters
+
+        // valid category
+        assertTrue(ItemCategory.isValidItemCategory("car")); //Normal case
+        assertTrue(ItemCategory.isValidItemCategory("essentials")); // Contains alphanumeric characters.
```

Comment:
Theres no numerical characters, since your category allows alphanumeric, maybe throw in some numbers in the string.